src/test/java/org/opensearch/plugin/insights/SearchSourceBuilderUtils.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.plugin.insights;
+
+import org.opensearch.index.query.BoolQueryBuilder;
+import org.opensearch.index.query.MatchQueryBuilder;
+import org.opensearch.index.query.QueryBuilders;
+import org.opensearch.index.query.RangeQueryBuilder;
+import org.opensearch.index.query.RegexpQueryBuilder;
+import org.opensearch.index.query.TermQueryBuilder;
+import org.opensearch.search.aggregations.bucket.terms.SignificantTextAggregationBuilder;
+import org.opensearch.search.aggregations.bucket.terms.TermsAggregationBuilder;
+import org.opensearch.search.aggregations.metrics.TopHitsAggregationBuilder;
+import org.opensearch.search.aggregations.pipeline.AvgBucketPipelineAggregationBuilder;
+import org.opensearch.search.aggregations.pipeline.DerivativePipelineAggregationBuilder;
+import org.opensearch.search.aggregations.pipeline.MaxBucketPipelineAggregationBuilder;
+import org.opensearch.search.aggregations.support.ValueType;
+import org.opensearch.search.builder.SearchSourceBuilder;
+import org.opensearch.search.sort.SortOrder;
+
+public class SearchSourceBuilderUtils {
+
+    public static SearchSourceBuilder createDefaultSearchSourceBuilder() {
+        SearchSourceBuilder sourceBuilder = new SearchSourceBuilder();
+        sourceBuilder.size(0);
+        // build query
+        TermQueryBuilder termQueryBuilder = QueryBuilders.termQuery("field1", "value2");
+        MatchQueryBuilder matchQueryBuilder = QueryBuilders.matchQuery("field2", "php");
+        RegexpQueryBuilder regexpQueryBuilder = new RegexpQueryBuilder("field3", "text");
+        RangeQueryBuilder rangeQueryBuilder = new RangeQueryBuilder("field4");
+        sourceBuilder.query(
```

Comment:
Can you add the structure of query being returned by each method in this class as comment? That will allow reviewers to quickly understand the logic

Reply:
Added json representation for each SearchSourceBuilder util